Sources/YapDB.swift

```diff
@@ -550,20 +556,20 @@ extension YapDB {
 
 extension YapDB {
 
-    public class Search {
-        public typealias Query = (searchTerm: String) -> String
+    open class Search {
```

Comment:
I prefer to leave these classes `open` so consumers of this pod can subclass them if that is useful for their application.